test/unit/content-src/components/DiscoveryStreamComponents/List.test.jsx

```diff
@@ -152,20 +145,6 @@ describe("<ListItem> presentation component", () => {
     assert.lengthOf(anchors, 1);
   });
 
-  it("should not contain 'span.ds-list-item-context' without props.context", () => {
-    const wrapper = shallow(<ListItem {...ValidListItemProps} />);
-
-    const contextEl = wrapper.find("span.ds-list-item-context");
-    assert.lengthOf(contextEl, 0);
-  });
-
-  it("should contain 'span.ds-list-item-context' spoc element", () => {
-    const wrapper = shallow(<ListItem {...ValidLSpocListItemProps} />);
-
-    const contextEl = wrapper.find("span.ds-list-item-context");
-    assert.lengthOf(contextEl, 1);
-  });
-
```

Comment:
We should bring this test in DSContextFooter essentially checking if we are setting story-sponsored-label when context is provided

Reply:
Spoke about keeping the test here and testing for SPOC in the DSContextFooter in List.

Upgraded test cases for Hero and DSCard to also test for Spoc!